### PR TITLE
zh-trans: add service-broker.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add service-broker.md

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
Basically LGTM. Leave it to @tengqm for another review.

File: content/zh/docs/reference/glossary/service-broker.md [hunk 1]
```diff
@@ -10,8 +24,20 @@ aka:
 tags:
 - extension
 ---
+-->
+
+<!--
  An endpoint for a set of {{< glossary_tooltip text="Managed Services" term_id="managed-service" >}} offered and maintained by a third-party.
+-->
+
+由第三方提供和维护的一组托管服务的端点。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/service-broker.md]: pls don't delete the glossary_tooltip shortcode.

**Pull request merged**